pages/ksphere/kommander/latest/centralized-monitoring/index.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: layout.pug
+navigationTitle: Centralized Monitoring
+title: Centralized Monitoring
+menuWeight: 7
+excerpt: Monitor clusters created with Kommander or any connected Konvoy cluster
+---
+
+Kommander provides centralized monitoring in a multi-cluster environment using the monitoring stack running in managed Konvoy clusters.
+Centralized monitoring is provided by default in every Kommander cluster.
+
+Managed Konvoy clusters are distinguished by a cluster ID.
+This cluster ID correlates to the kube-system namespace UID of the cluster.
+You can run this kubectl command, **using the correct cluster's context or kubeconfig**, to look up the cluster's kube-system namespace UID to determine which cluster the metrics and alerts correspond to:
+
+```bash
+$ kubectl get namespace kube-system -o jsonpath='{.metadata.uid}'
+```
+
+## Centralized Metrics
+
+Metrics are aggregated from all managed Konvoy clusters up to the Kommander cluster using Thanos.
+You can visualize these metrics in Grafana using a set of provided dashboards.
+The [Thanos Query][thanos_query] component is installed on the Kommander cluster and set up to query the Prometheus instances running in the managed clusters via a Thanos sidecar running alongside each Prometheus container.
+Grafana is configured with Thanos Query as its datasource, and comes with a pre-installed dashboard for a global view of all managed clusters, named `Kubernetes / Compute Resources / Clusters `.
+The `Thanos Query` dashboard is also installed by default to monitor the Thanos Query component.
+
+You can access the centralized Grafana UI at `https://<CLUSTER_URL>/ops/portal/kommander/monitoring/grafana`.
```

Comment:
nit: is it possible to put all the URLs in their own box when you render the markup? That way it is easier to spot these URLs